src/plugins/notebook/pluginSpec.js

```diff
@@ -211,10 +211,17 @@ describe("Notebook plugin:", () => {
 
         describe("synchronization", () => {
 
+            let objectCloneToSyncFrom;
+
+            beforeEach(() => {
+                objectCloneToSyncFrom = JSON.parse(JSON.stringify(notebookViewObject));
```

Comment:
Do we want to use [structuredClone](https://developer.mozilla.org/en-US/docs/Web/API/structuredClone) here? It looks pretty well supported now.

Reply:
Was unaware of this function, cool, thanks! I'll change it.